_docs/for-contributors/2017-01-02-implementation.md

```diff
@@ -190,6 +190,29 @@ Broadcast messages are resent to neighbors right after retrieval (before
 handling). Also, they are being checked against LRU cache, and messages
 that have been already received once get ignored.
 
+### Leaders and rich men computation (LRC)
+"Slot leaders" and "rich men" are two important notions of Ouroboros Proof of Stake Algorithm:
+
+- Slot leaders: A slot leader is a such node which creates block in the current slot.
+Slot leaders for the current epoch (for each slot of the current epoch) are computed by [Follow the Satoshi](/cardano/proof-of-stake/#follow-the-satoshi) (FTS) algorithm in the beginning of current epoch.
+FTS uses a `shared seed` which is result of [Multi Party Computation](/cardano/proof-of-stake/#multi-party-computation) (MPC) algorithm for previous epoch: in the result of MPC some nodes reveal their seeds, `xor` of these seeds is called `shared seed`.
+
+- Rich men: Only nodes which sent their VSS certificates
+and also has ehought stake can participate in MPC algorithm. 
```

Comment:
`enought` — typo
Also, is it described somewhere what is enough? Like some threshold, where it is defined an described?